openbas-front/src/actions/injects/Inject.d.ts

```diff
@@ -9,9 +9,8 @@ export type InjectInput = {
   inject_depends_duration_seconds: number;
 };
 
-export type InjectStore = Omit<Inject, 'inject_tags' | 'inject_content' | 'inject_injector_contract' | 'inject_teams'> & {
+export type InjectStore = Omit<Inject, 'inject_tags' | 'inject_content' | 'inject_injector_contract'> & {
```

Comment:
Why are you removing this ?

Reply:
I need to access inject_teams to enable/disable the Test action